internal/pkg/iptablesutils/iptables.go

```diff
@@ -119,10 +119,7 @@ func findMatchingEntries(k0sBinPath, mode string, entries ...string) (entriesFou
 
 	v4Err, v6Err := findMatches("iptables-save"), findMatches("ip6tables-save")
 	if v4Err != nil && v6Err != nil {
-		return false, 0, multierr.Combine(
-			fmt.Errorf("iptables-save: %w", v4Err),
-			fmt.Errorf("ip6tables-save: %w", v6Err),
-		)
+		return false, 0, fmt.Errorf("iptables-save: %w; ip6tables-save: %w", v4Err, v6Err)
```

Comment:
There's a `;` again, are these intentional? :)

Reply:
I suppose they are. Looks a bit confusing to me.

Reply:
What would be a more appropriate separator? A newline, as in `errors.Join(...)`?